src/pages-conditional/wallets.tsx

```diff
@@ -331,6 +336,7 @@ const WalletsPage = ({
                 emoji={type.emoji}
                 description={type.description}
                 emojiSize={2.5}
+                alignItems="center"
```

Comment:
This should also be fixed inside of the `HorizontalCard` component.

Reply:
thanks for these @pettinarip